nreader/src/EUTelNativeReader.cc

```diff
@@ -145,7 +147,14 @@ void EUTelNativeReader::readDataSource(int numEvents) {
   }
 
   if ( reader->Event().IsBORE() ) {
-    eudaq::PluginManager::Initialize(  reader->Event() );
+//    eudaq::PluginManager::Initialize(  reader->Event() );
+    
+    const eudaq::DetectorEvent & ev = reader->Event();
+    string config = ev.GetTag<std::string>("CONFIG","Config not found");
+    size_t dutPosPlace = config.find("DUTposition = ");
+    string dutPosStr = config.substr(dutPosPlace+14,3);
```

Comment:
This was a temporary fix curing a short coming in our producer for the pALPIDEfs. I've removed this lines again in a new commit